Wallbox documentation (related to HA PR 48082) (#17085)
home-assistant/home-assistant.github.io: source/_integrations/wallbox.markdown

---
title: Wallbox
description: Instructions on how to integrate sensors Wallbox EV chargers to Home Assistant
ha_category:
  - Car
ha_release: 2021.4
ha_iot_class: Cloud Polling
ha_domain: wallbox
ha_platforms:
  - sensor
ha_config_flow: true
---

> This `wallbox` sensor platform pulls data from https://my.wallbox.com for your wallbox charging station.

Reviewer: ```suggestion
The Wallbiox integration pulls data from the [MyWallbox Portal](https://my.wallbox.com) for your Wallbox charging station.
```

{% include integrations/config_flow.md %}

## Sensors

> The integration adds the following sensors:

Reviewer: ```suggestion
The integration adds the following sensors:

```

---
title: Wallbox
description: Instructions on how to integrate sensors Wallbox EV chargers to Home Assistant
ha_category:
  - Car

> ha_release: 2021.4

Reviewer: ```suggestion
ha_release: 2021.6
```

---
title: Wallbox
description: Instructions on how to integrate sensors Wallbox EV chargers to Home Assistant
ha_category:
  - Car
ha_release: 2021.4
ha_iot_class: Cloud Polling
ha_domain: wallbox
ha_platforms:

>   - sensor

Reviewer 2: Add here `ha_config_flow: true`

---

This `wallbox` sensor platform pulls data from https://my.wallbox.com for your wallbox charging station.

> ## Configuration
> 
> To enable this sensor in your installation, go to Configuration and Integrations and add a MyWallbox integration. You will need to fill in your wallbox:
> - serial number
> -  your username (email)
> -  password

Reviewer 2: ```suggestion
{% include integrations/config_flow.md %}
```

Author: @klaasnicolaas, thnx for the review. I included your proposals
